script/Ciel/Day 12/12_01_CIEL12_3_2A.txt

```diff
@@ -236,30 +254,33 @@ On this night, as tranquil as any other, in this mansion that no one ever comes
 [sha:39a1e0f606e899690f25b79686a1dfe02fa857af]{
 -- Page 191, Offset 31393.
 -- 何かを欠いたまま沈んでいく。
-I sink back down, plagued by some sort of deficiency.
+I sink into the depths,
+ still missing something I can't identify.
 }
 [sha:bf626454db9d63cfe830e37a0997d8a4327a0b67]{
 -- Page 192, Offset 31394.
 -- 目蓋を閉じる。
-I close my eyelids.
+I close my eyes.
 }
 [sha:281fa17bac5a8e6e5a48228a5f0c6b5a2f3131d5]{
 -- Page 192, Offset 31395.
 -- 頭痛から遠ざかる。
-I feel the headache growing distant.
+My headache recedes into the distance.
 }
 [sha:8422e4486a80a2303b3992f5b463a7267512c788]{
 -- Page 192, Offset 31396.
 -- 海の真ん中で浮き輪から手を離したような水没感。
-A feeling of submersion as if releasing my grip on a float in the middle of the ocean.
+I feel myself slip beneath the surface,
+ as though I've just released my grip on a lifebuoy bobbing in the middle of the ocean.
```

Comment:
Man, this sentence sounds weird in English. The first clause feels like an indirect metaphor, while the second is a direct comparison; they don't quite add up. Even so, it's a tricky one, because 水没感 is supposed to sound sort of pleasant, while the comparison is more ominous.
It might need some dramatic changes.

I feel a bit unsure, but here's an attempt, using some contrasting adjectives.
>I submit myself to the tranquil depths, letting go of the lifebuoy keeping me afloat in the middle of the raging oceans.